admin/class-acc_user_importer-admin.php

```diff
@@ -319,6 +322,19 @@ private function proccess_user_data ( $users ) {
 				continue;
 			}
 
+			switch($loginNameMapping) {
+				case 'ContactId':
+					$loginName = $userContactId;
+					break;
+				case 'firstname-lastname':
+					$loginName = "$userFirstName-$userLastName";
+					break;
+				case 'ImisId':
```

Comment:
Could we use the syntax 
     case 'imis_id' which would be exactly what the ACC Rest API exports?  Easier to read too, I find.

Reply:
We can, the inconsistency is really bothering me so I changed it :joy: Your argument makes sense though, I'll change it.